dev/integration_tests/new_gallery/lib/pages/home.dart

```diff
@@ -1166,47 +1164,8 @@ class _StudyWrapperState extends State<StudyWrapper> {
               child: widget.study,
             ),
           ),
-          if (!isDisplayFoldable(context))
```

Comment:
This is almost certainly the mistake - given that the conditional is `!isDisplayFoldable(context)` I shouldn't have deleted the contents, but rather left them around. Will test and put up a new pr if that's correct